ios-trivia-game/ViewControllers/ResultsViewController.swift

```diff
@@ -32,4 +33,15 @@ class ResultsViewController: UIViewController {
     }
     */
 
+    @IBAction func onQuitFromResults(_ sender: UIBarButtonItem) {
+        // update user_in_game to remove player from user_in_game
+        
+        FirebaseClient.instance.quitGame(complete: {
```

Comment:
Seems like this is an occurring function that's being used a lot. So we can consider putting this into a separate function so we don't end up rewriting this every time we need to quit the game.